cluster/node-pools/master-default/stack.yaml

```diff
@@ -49,6 +49,9 @@ Resources:
     Properties:
       LaunchTemplateName: '{{.Cluster.LocalID}}-{{ .NodePool.Name }}'
       LaunchTemplateData:
+        MetadataOptions:
+          HttpTokens: required
+          HttpPutResponseHopLimit: 2
```

Comment:
see also https://github.com/aws/elastic-beanstalk-roadmap/issues/242 for example

Reply:
The idea was to _prevent_ that pods can call the metadata endpoint to avoid that they can get instance credentials, that's why I didn't set this to leave it at `1`. This currently does not work because kube-node-ready depend on metadata service, but this would give it access to the instance credentials from my understanding which is not intended.

Reply:
I thought that the delegation requires the hop limit to set to 2 anyways, but the definition is not clear to me (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-metadataoptions.html):

> HttpPutResponseHopLimit
    The desired HTTP PUT response hop limit for instance metadata requests. The larger the number, the further instance metadata requests can travel.